AdminDashboard/BusinessLogicOrchestrators/AccountOrchestrator/AccountOrchestrator.cs

```diff
@@ -0,0 +1,56 @@
+﻿using AdminDashboard.Main.Builders;
+using AdminDashboard.Models.Domain;
+using AdminDashboard.Models.EndUserRequests;
+using AdminDashboard.Models.JsonRequests;
+using AdminDashboard.Repositories.AccountRepository;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AdminDashboard.BusinessLogicOrchestrators.AccountOrchestrator
+{
+    public class AccountOrchestrator : IAccountOrchestrator
```

Comment:
Orchestrators handle all the high level business logic and delegate lower level details to the database. Also, orchestrators communicate with the business domain models!